### DISCUSSION TITLE
Multiple SP support?

### DISCUSSION THREAD

**Discussion by cch99** [codegreencreative/laravel-samlidp#123] [General]
Thanks for developing this module.  It's exactly what I'm looking for, maybe.  Does laravel-samlidp support multiple SPs?  If so is there a sample config showing how to set this up?

Thanks again!

**Comment by upwebdesign** [comment]
@cch99 See the config section https://github.com/codegreencreative/laravel-samlidp#config. You can add many SP's in the samlidp.config file by adding to the 'sp' array.
